src/authentication/authentication.module.ts

```diff
@@ -22,9 +23,10 @@ import {AdminsService} from '../admins/admins.service';
         expiresIn: environment.JWT_TOKEN_LIFE,
       },
     }),
+    UsersModule,
   ],
   controllers: [AuthenticationController],
-  providers: [AuthenticationService, AppStrategy, ...userProviders, UsersService, ...adminProviders, AdminsService, AdminStrategy],
+  providers: [AuthenticationService, AppStrategy, ...adminProviders, AdminsService, AdminStrategy],
```

Comment:
Instead of directly providing the UserService, the UsersModule is now imported inside this module.
but I'm not sure how to check that everything works well in this module, so I need help with this.

Reply:
Sounds right, after fixing everything I will help verify it